ServerCore/Areas/Identity/Pages/Account/ExternalLogin.cshtml.cs

```diff
@@ -120,12 +122,19 @@ public async Task<IActionResult> OnPostConfirmationAsync(string returnUrl = null
 
                         if (ModelState.IsValid)
                         {
+                            // If there are no GlobalAdmins make them the GlobalAdmin
+                            if ((Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") == EnvironmentName.Development) && !_context.PuzzleUsers.Where(u => u.IsGlobalAdmin).Any())
```

Comment:
Is there any chance of a random bot becoming global admin of our devtest site?

Reply:
Only if they can sign in with MSA